roles/bridge-setup/tasks/main.yml

```diff
@@ -1,9 +1,31 @@
 ---
 
-- name: Install bridge-utils
+- name: Install bridge-utils and iptables
   yum:
-    name: bridge-utils
+    name: "{{ item }}"
     state: latest
+  with_items:
```

Comment:
This is actually a really slow method I learned about recently. You should move away from the `with_items` method, and just pass as a comma separated list.

```
  name: bridge-utils,iptables,iptables-services
```

Reply:
changed.